### PR TITLE
Use yum repos for base image creation

### PR DESCRIPTION
**Pull request by lcarva**

### CODE REVIEW

File: atomic_reactor/plugins/pre_add_filesystem.py [hunk 1]
```diff
@@ -137,6 +152,8 @@ def parse_image_build_config(self, config_file_name):
         section = 'image-build'
         for option in ('name', 'version', 'arches', 'target', 'install_tree'):
             value = config.get(section, option)
```
Comment (vrutkovs) [inline atomic_reactor/plugins/pre_add_filesystem.py]: configparser would throw ```NoOptionError``` if the option is not specified. Either we should catch it or verify with  ```config.has_option```

**Comment by vrutkovs** [comment]
LGTM

**Comment by twaugh** [comment]
LGTM too

File: atomic_reactor/plugins/pre_add_filesystem.py (hunk 1 above)
Comment (lcarva) [inline atomic_reactor/plugins/pre_add_filesystem.py]: It still would, but we're building a default config as an input to config parser which is dynamically generated. This check is to ensure that neither the config that was dynamically generated nor the config provided by user has somehow provided an empty value, such as:

```
install_tree =
```

**Comment by lcarva** [comment]
Tested.

**Pull request merged**